tests/test_include_markdown.py

```diff
@@ -87,6 +87,31 @@
 
 This must be included.
 
+<!-- END INCLUDE -->
+''',
+        ),
+
+        # Start and end arguments with escape sequence
+        (
+            '''# Header
+
+{%
+  include-markdown "{filepath}"
+  start="<!--\\tstart -->"
+  end="<!--\\tend -->"
+%}
+''',
+            '''This must be ignored.
+<!--\tstart -->
+This must be included.
+<!--\tend -->
+This must be ignored also.''',
+            '''# Header
+
+<!-- BEGIN INCLUDE {filepath} &lt;!--\tstart --&gt; &lt;!--\tend --&gt; -->
+
+This must be included.
+
 <!-- END INCLUDE -->
 ''',
         )
```

Comment:
Can you add another test with special characters unescaped? I'm thinking on:


```python
        # Unescaped special characters
        (
            '''# Header

{%
  include-markdown "{filepath}"
  start="<!--\nstart -->"
  end="<!--\nend -->"
%}
''',
            '''This must be ignored.
<!--\nstart -->
This must be included.
<!--\nend -->
This must be ignored also.''',
            '''# Header

<!-- BEGIN INCLUDE {filepath} &lt;!--\nstart --&gt; &lt;!--\nend --&gt; -->

This must be included.

<!-- END INCLUDE -->
''',
        )
```

Reply:
Roger, will do.